front_end/server/lib/routes/servers.js

```diff
@@ -1,4 +1,19 @@
 const express = require('express');
 const router = express.Router();
 const Server = require('../models/server');
-const bodyParser = require('body-parser').json();
\ No newline at end of file
+const bodyParser = require('body-parser').json();
+
+router
+  .get('/', (req, res, next) => {
+    Server.find()
+      .lean()
+      .then(servers => res.send(servers))
+      .catch(next);
+  })
+  .post('/', bodyParser, (req, res, next) => {
+    new Server(req.body).save()
```

Comment:
oh wow ok so this is actually doing quite a lot! is newServer the document that was just saved? I guess Im confused by the name.  Is document what is stored in (Im assuming) mongodb?